### ISSUE TITLE
Future Development roadmap

### ISSUE THREAD

**Issue by fbaldo31** [realtymaps/promise-sftp#16]
Hi i would like to know if it is planned to add :

- [ ] ascii
- [ ] binary

I need soon theses features, so let me know if you want contributions on these subjects ?

**Comment by zacronos** [comment]
Yes, contributions would be greatly appreciated.  I do not currently have a project which uses this library, so I am no longer actively developing.  I will continue to maintain and handle PRs, however.
